website/content/gateway/api_reference/resources/authorizations/authentications/3dsecure.md

```diff
@@ -8,16 +8,18 @@ weight: 140
 
 Only one 3-D Secure version can be used for a given authorization.
 
-{{% description_list %}}
-{{% description_term %}}[3dsecure][v1] {{% regex %}}dictionary{{% /regex %}}{{% /description_term %}}
-{{% description_details %}}3-D Secure version 1.
-{{% regex_optional %}}Optional. Cannot be present if `v2` is present.{{% /regex_optional %}}{{% /description_details %}}
-
 {{% description_term %}}[3dsecure][v2] {{% regex %}}dictionary{{% /regex %}}
 {{% /description_term %}}
 {{% description_details %}}
 3-D Secure version 2, also known as EMV 3-D Secure.
 {{% regex_optional %}}Optional. Cannot be present if `v1` is present{{% /regex_optional %}}
```

Comment:
```suggestion
{{% regex_optional %}}Optional. ~~Cannot be present if `v1` is present.~~{{% /regex_optional %}}
```

Reply:
I thought about this, and decided not to include it — but to easily notice what should be removed, you are very right. I'll include this 👍